Content.Shared/Humanoid/Markings/MarkingsSet.cs

```diff
@@ -201,9 +202,11 @@ public void EnsureValid(MarkingManager? markingManager = null)
     /// <summary>
     ///     Ensures that the default markings as defined by the marking point set in this marking set are applied.
     /// </summary>
-    /// <param name="skinColor">Color to apply.</param>
+    /// <param name="skinColor">Skin color for marking coloring.</param>
+    /// <param name="eyeColor">Eye color for marking coloring.</param>
+    /// <param name="hairColor">Hair color for marking coloring.</param>
```

Comment:
im with flipp here that im not really a fan of this hardcoding eye/hair/facialhair stuff into markings and humanoid any more than it already is, though i dont know enough about the underlying system to block this PR over it if @vulppine doesnt have an issue with it

Reply:
coming back to this again, it makes me wonder how difficult it would be to move these marking default modes that require a hard-coded hair color into its own thing where you fetch a default color for the marking based on whatever color you can get from that marking set category (i.e., fetching the first marking's first color) and otherwise defaulting to something like `Color.Black` or whatever), but i think eye color might be out of the question since that isn't necessarily a marking

Reply:
maybe I can try give it an humanoid data instead of hair and eyes color